examples/quick_start.ipynb

```diff
@@ -0,0 +1,260 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zDk-e7Pg_EdD"
+      },
+      "source": [
+        "# Quick start\n",
+        "\n",
+        "`optax` is a simple optimization library for Jax. The main object is the `GradientTransformation` which can be chained\n",
+        "with other transformations to obtain the final update operation and optimizer state.\n",
+        "`optax` also contains some simple loss functions and utilities to help you write the full optimization steps. This colab walks through a few examples on how to use optax to train a simple linear model:\n",
+        "\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Jr7_e_ZJ_hky"
+      },
+      "outputs": [],
+      "source": [
+        "import jax.numpy as jnp\n",
+        "import jax\n",
+        "import optax\n",
+        "import functools"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "n7kMS9kyM8vM"
+      },
+      "source": [
+        "We first setup a simple linear model and a loss function. You can use any other library like [haiku](https://github.com/deepmind/dm-haiku) to construct your networks. Here we keep it simple and write it ourselves. We use `l2_loss` from optax's [common loss functions](https://optax.readthedocs.io/en/latest/api.html#common-losses)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "0-8XwoQF_AO2"
+      },
+      "outputs": [],
+      "source": [
+        "@functools.partial(jax.vmap, in_axes=(None, 0))\n",
+        "def network(params, x):\n",
+        "  return jnp.dot(params, x)\n",
+        "\n",
+        "def compute_loss(params, x, y):\n",
+        "  y_pred = network(params, x)\n",
+        "  loss = jnp.mean(optax.l2_loss(y_pred, y))\n",
+        "  return loss"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EZviuSmuNFsC"
+      },
+      "source": [
+        "Create some data to learn from"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "H-_pwBx6_keL"
+      },
+      "outputs": [],
+      "source": [
+        "key = jax.random.PRNGKey(42)\n",
+        "target_params = 0.5\n",
+        "\n",
+        "# Generate some data.\n",
+        "xs = jax.random.normal(key, (16, 2))\n",
+        "ys = jnp.sum(xs * target_params, axis=-1)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Td4Lp3qDNsL3"
+      },
+      "source": [
+        "## Basic usage of `optax`\n",
```

Comment:
Suggestion: I think it's Optax (name) vs package name optax (https://github.com/deepmind/optax/blob/master/README.md)

```suggestion
        "## Basic uses of Optax\n",
```